### PR TITLE
(#370) Rename Chocolatey's Azure Marketplace offer

### LINKED ISSUE #370: Chocolatey's Azure Marketplace offer has been renamed
Chocolatey's Azure Marketplace offer has been renamed and the docs need to be updated to reflect this.

This will include renaming some pages, and may include moving the section to another parent.

### PR DESCRIPTION
**Pull request by Windos**
## Description Of Changes

This PR reflects the updated name for Chocolatey's Azure Marketplace offer to "Chocolatey for Business Azure Environment" and drops the use of the "QDE" from the name.

It also moves this section of documentation out from under the "Quick Deployment Environment (QDE)" parent path.

## Motivation and Context

The official name for the offer/environment has changed and needs to be reflected wherever it is referenced.

## Testing

This has been tested by previewing the site after changes have been made. I've verified that all changes render properly, and updated xref's work.

Screenshot of previewed changes

Redirects from old URLs have been tested using the preview too.

## Change Types Made

* [x] Bug fix (non-breaking change)
* [ ] Feature / Enhancement (non-breaking change)
* [ ] Breaking change (fix or feature that could cause existing functionality to change)
* [ ] PowerShell code changes.

## Related Issue

Fixes #370 

## Change Checklist

* [ ] Requires a change to the documentation
* [x] Documentation has been updated
* [ ] Tests to cover my changes, have been added
* [x] All new and existing tests passed.
* [ ] PowerShell v2 compatibility checked.

### CODE REVIEW

File: input/en-us/quick-deployment/azure/client-setup.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-Description: How to setup a client machine to use Chocolatey for Business QDE Azure Environment
+Description: How to setup a client machine to use Chocolatey for Business Azure Environment
 ---
 
 > :memo: **NOTE**
 >
-> This document is for the Chocolatey for Business QDE Azure Environment from the [Azure Marketplace](https://ch0.co/azureqde).
-> If you are using a non-Azure version of QDE, please refer to the [QDE v2 Client Setup page](xref:v2-client-setup).
+> This document is for the Chocolatey for Business Azure Environment from the [Azure Marketplace](https://ch0.co/C4B-Azure).
+> If you are using the Quick Deployment Environment, please refer to the [QDE v2 Client Setup page](xref:v2-client-setup).
```
Comment (Windos) [inline input/en-us/quick-deployment/azure/client-setup.md:11]: Question for @pauby & @JPRuskin - Do we still need this note?

File: input/en-us/c4b-environments/azure/index.md [hunk 2]
```diff
@@ -1,85 +1,84 @@
 ---
-Order: 25
-xref: qdeazure
-Title: Chocolatey for Business QDE Azure Environment
-Description: High level information about the Chocolatey for Business QDE Azure Environment
+Order: 50
```
Comment (gep13) [inline input/en-us/c4b-environments/azure/index.md:2]: Is there a reason that this is 50? This index.md file is the only one within the current node, so this doesn't need to have value so high.

**Comment by gep13** [comment]
@Windos on the whole, this looks good to me.  It will need to be rebased on the current head of master before it can be merged.

I have left one comment/question, and it looks like there is an open question to @pauby and @JPRuskin, does this need to be answered before moving forward?

Just to confirm one thing as well.  Was it intentional to put this new node here:

image

Before the open source offerings?  Just wanted to validate this.

File: input/en-us/c4b-environments/azure/index.md (hunk 2 above)
Comment (Windos) [reply input/en-us/c4b-environments/azure/index.md]: I dropped the order down to 20. I imagine Azure will always be the first item under the parent, but this leaves space for something else if I'm wrong.

Comment (gep13) [reply input/en-us/c4b-environments/azure/index.md]: I normally put the first page in a folder at 10, and then every subsequent page in increments of 10.  This means that we can slot in a couple of new pages, without having to change the orders on a bunch of other files.  Having this a 20 is fine, no issues there.

**Review by gep13 — DISMISSED** [review]
LGTM!

**Comment by gep13** [comment]
@pauby these changes all look good to me.  Just as a final sanity check though, based on previous conversations, can you confirm that this is the menu structure that you want:

image

**Comment by pauby** [comment]
This isn't the structure we should have.

I think this should go where QDE is, ie. near the bottom. We can then move QDE inside that menu structure at some point. I think it also makes sense for the QSG bits to go in there too.

File: input/en-us/quick-deployment/azure/client-setup.md (hunk 1 above)
Comment (Windos) [reply input/en-us/quick-deployment/azure/client-setup.md]: Decision made to remove this note.

**Comment by pauby** [comment]
Thanks @windos, for updating that. I'm happy.

**Comment by gep13** [comment]
> @ryanrichter94 said...
You'll wanna update the ch0.co short link when this gets deployed.

That shouldn't need to happen immediately, as the necessary redirects should be in place within this PR.  i.e. any change in structure uses a `RedirectFrom` to ensure that folks get to the right place.

Updating the shortlink to point to the actual location makes sense to do, but it shouldn't be a requirement before deployment.

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@Windos thanks for getting this updated!

**Pull request merged**